README.md

```diff
@@ -229,6 +229,26 @@ export const globalConfig = {
 };
 ```
 
+## Using default parameters
+
+There might be situations where default parameters are needed, for example when using a language in a URL. These default parameters can be set with the `defaultParams` object in your [endpointConfig](https://github.com/oberonamsterdam/react-api-data#config):
+
+```js
+const endpointConfig = {
+    getData: {
+        url: `${BASE_URL}/:language/getData.json`,
+        method: 'GET',
+        defaultParams: {
+            language: 'en',
+        },
+    },
+};
+```
+
+Multiple default params can be set. Their value should be either a `string` or a `number`.
```

Comment:
dit klinkt nu alsof je voor bijvoorbeeld language twee verschillende defaultparams tegelijk kan instellen, terwijl je bedoelt dat je voor meerdere params tegelijk een default waarde kunt instellen.

Je zou iets kunnen zeggen in de trand van:
You can set default params for multiple params or only part of the params. Their value should be either a `string` or a `number`.